requirements.txt

```diff
@@ -0,0 +1,17 @@
+aiohttp==3.7.4.post0
+async-timeout==3.0.1
+attrs==21.4.0
+chardet==4.0.0
+click==8.1.2
+discord.py==1.7.3
+idna==3.3
+multidict==6.0.2
+mypy-extensions==0.4.3
+networkx==2.7.1
+numpy==1.22.3
+pathspec==0.9.0
+platformdirs==2.5.1
+tomli==2.0.1
+typing-extensions==4.1.1
+yapf==0.32.0
+yarl==1.7.2
```

Comment:
Why do we so much requirements? It seems easier to just install the top-level ones.

Reply:
Not sure why there's so much stuff

I just created a virtual environment and then installed discord.py, numpy, networkx, and black and then did a `pip freeze >> requirements.txt`

We _can_ probably manually remove some of this but I'm inclined to just use whatever pip spits out

Reply:
I would prefer to leave just what you installed with pip and not even include versions. At least in the develpment phase.

So `requirements.txt` would look like:
```
numpy
networkx
black
```